### ISSUE TITLE
Wall Panel closing/ crashing after some time

### ISSUE THREAD

**Issue by Madseason41** [thanksmister/wallpanel-android#49]
**Describe the bug**
Open wall panel, settings include:
Prevent device sleep and camera streaming enabled. Wait and eventually the tablet will be a blank screen. I think this is camera streaming related as I just started using this feature. Both wall panel versions seemed to not crash with camera streaming disabled. This is a guess though.

**To Reproduce**
1. Start Wall panel
2. Configure settings (prevent device sleep, and camera streaming enabled)
3. Wait a few hours and the amazon Fire 7 will be a blank/black screen.
4. Fix by pressing power button and wake up tablet, click on wall panel to reopen it.

**Expected behavior**
Device should never revert to a blank screen

 - Device: Amazon Fire 7 Tablet
 - OS: 5.3.6.4
 - Browser: stock
 - Version: 8.5.5 (tried on 8.5.1 as well)

**Comment by thanksmister** [comment]
Update to the latest wall panel version.

**Comment by Madseason41** [comment]
I’m on the latest, and have tried on 8.5.1 as well. 

Battery is at 100% and it’s power by POE/ USB adapter.

**Comment by thanksmister** [comment]
I am testing, I have version 8.5.1 FireOS.

**Comment by Madseason41** [comment]
I think I had a typo on OS version number. I checked for updates this morning and the device said I was on the latest. I think it’s 5.6.3.4, not 5.3.6.4. I will double check when I get home.

**Comment by thanksmister** [comment]
My tests on a new Fire Table running 8.5.1 stock showed no issues.  I was using camera streaming and wake on motion.  I also used the built-in clock screensaver feature and prevent screen sleep.

**Comment by Madseason41** [comment]
Ok just updated to 8.5.6 wall panel version. I have my settings set to the same as above. I will continue to keep an eye on it. I think it may be tied to calendar alerts in some manner and I am going to mess with that and see what I can figure out. Thanks for the help.

**Comment by Madseason41** [comment]
Recommend closing. I think in developer options, the setting "stay awake" was reset. I turned it back on and all seems ok now. Thanks for the help

**Comment by thanksmister** [comment]
But why must you use the stay awake option?   The screensaver already prevents sleep and there is a prevent sleep option.

**Comment by Madseason41** [comment]
After my last reply I got to thinking about that as well. Something still didn’t seem right, the screensaver never would appear when I selected it.  So I just chose the nuclear option and reset to factory settings. Selected allow install from unknown sources (no other selections were made), rebooted, downloaded latest version from github and installed. NOW it seems to be working correctly. Screen saver appears and motion detection works as expected. I recommend closing this issue as I am pretty sure it was switchology on my device. Thanks again.
